src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java

```diff
@@ -39,42 +40,36 @@
 import java.lang.ref.WeakReference;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.UUID;
 
 /**
  * Holds logic for an active pagination that is occurring.
  */
-abstract class ActivePagination {
+class ActivePagination {
```

Comment:
@zml2008 Unfortunately, the Unicode arrows didn't work when I tried it. In addition, using an Iterable makes the per-page stuff harder.